### PR TITLE
Fixes issues with Decapitation and NODROP items

### LINKED ISSUE #18430: Cluwne mask becomes invisible after decapitation
**Issue Description**:
A cluwne's cursed clown mask becomes invisible if the cluwne is surgically decapitated and has their head reattached. They're still "wearing" the mask (which can be seen upon inspection), but it doesn't physically appear on their face.
image

**What did you expect to happen**: 
For the cluwne mask to reappear once the cluwne is revived, since his torment is inescapable

**What happened instead**:
The cluwne mask did not reappear

**Why is this bad/What are the consequences**:
Cluwning is meant to be irreversible and unconcealable 

**Steps to reproduce the problem**:
1. Cluwne someone
2. Surgically decapitate them
3. Reattach their head
4. Revive them
5. No mask!

**When did the problem start happening**:
N/A

**Extra information**:
honk

### LINKED ISSUE #18612: Eye of God remains in eye slot after decapitation
**Issue Description**:

The Eye of God remains in the body's eyes slot after the head is removed.

**What did you expect to happen**: 

Head removed = no longer attached to body. Eyes attached to head. Eye of God in eyes. Not attached to body.

**What happened instead**:

Eye of God does not decouple with the head and instead remains staunchly stuck in the body's eye slot.

**Why is this bad/What are the consequences**:

Makes no sense. Should be taken with the head, and be able to be cut out with a sharp object like the ears and brain organs.

**Steps to reproduce the problem**:

Step 1: Acquire Eye of God
Step 2: Equip Eye of God
Step 3: Unequip your head
Step 4: Magical floaty eyes

**When did the problem start happening**:

Probably since Eye of God was added?

**Extra information**:

Issue was reported by Tourte, yell at him for stuff

### PR DESCRIPTION
**Pull request by Sirryan2002**
## What Does This PR Do
Forces items in slots held by decapitated limbs to force drop.

Also makes it so that eye of god can be combined properly in respects to its NODROP functionality.

Gives the Cluwne mask del on drop flag so that it deletes itself if a cluwne gets decapitated, 

Fixes #18430 technically since this prevents this from ever happening
Fixes #18612

## Why It's Good For The Game
The cluwne thing is less of a fix and more of a design thing as I don't think people should be able to cluwne themselves if they somehow manage to get the mask off a cluwne. It's a round-ending action if they wear it, it's best if cluwne masks on the floor are only available by intentional admin spawn.

Otherwise NODROP items, while NODROP shouldn't stay in slots that don't have limbs anymore. If the item isn't meant to be picked up, the Abstract or DEL_DROP flags should handle that fine.

## Testing
Confirmed that all items drop or delete correctly upon decapitation/gibbing.
Eye of god confirmed to combine correctly. However testing has revealed that there is other issue in regards to updating clothing overlays as the proc used to update them doesn't seem to do their job, the current implementation with the eye of God is fine however fixing the overlay issue is out of scope.

## Changelog
:cl:
fix: Fixed eye of god combining incorrectly
fix: Fixed NODROP masks not removing themselves correctly upon the user losing their head
/:cl:

### CODE REVIEW

**Comment by Inadvizable** [comment]
Shamebrero nerf.

File: code/modules/surgery/organs/subtypes/standard.dm [hunk 1]
```diff
@@ -234,15 +234,15 @@
...
+			owner.unEquip(owner.head, TRUE)
 		if(owner.l_ear)
-			owner.unEquip(owner.l_ear)
+			owner.unEquip(owner.l_ear, TRUE)
 		if(owner.r_ear)
-			owner.unEquip(owner.r_ear)
+			owner.unEquip(owner.r_ear, TRUE)
 		if(owner.wear_mask)
-			owner.unEquip(owner.wear_mask)
+			owner.unEquip(owner.wear_mask, TRUE)
```
Comment (hal9000PR) [inline code/modules/surgery/organs/subtypes/standard.dm:245]: would you name the TRUE arguement? makes it slightly more readable.

Comment (Sirryan2002) [reply code/modules/surgery/organs/subtypes/standard.dm:245]: yes

Comment (Sirryan2002) [reply code/modules/surgery/organs/subtypes/standard.dm]: done

File: code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -478,19 +478,21 @@
...
+	var/double_eye = FALSE
+
+/obj/item/clothing/glasses/godeye/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/clothing/glasses/godeye) && W != src)
+		var/obj/item/clothing/glasses/godeye/second_eye = W
+		if(!second_eye.double_eye && !double_eye)
+			double_eye = TRUE
+			icon_state = "doublegodeye"
+			item_state = "doublegodeye"
+			desc = "A pair of strange eyes, said to have been torn from an omniscient creature that used to roam the wastes. There's no real reason to have two, but that isn't stopping you."
```
Comment (SteelSlayer) [inline code/modules/clothing/glasses/glasses.dm:491]: Minor thing, but could you also change their name to `eyes of god`? Since it's a pair.

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**